.devcontainer/Dockerfile

```diff
@@ -0,0 +1,5 @@
+FROM --platform=linux/amd64 mcr.microsoft.com/devcontainers/python:1-3.11-bullseye
```

Comment:
```suggestion
FROM --platform=linux/amd64 mcr.microsoft.com/devcontainers/python:1-3.11-bullseye
# We need to force the container to be amd so that it works on a Mac. Without this the functions extension doesn't install.
```